packages/admin/src/table/filterbar/filterBarButton/FilterBarButton.styles.tsx

```diff
@@ -0,0 +1,60 @@
+import { createStyles, Theme } from "@material-ui/core";
+
+import { FilterBarButtonProps } from "./FilterBarButton";
+
+export type FilterBarButtonClassKey = "root" | "open" | "hasDirtyFields" | "filterBadge";
+
+export const styles = (theme: Theme) => {
+    return createStyles<FilterBarButtonClassKey, FilterBarButtonProps>({
+        root: {
+            position: "relative",
+            alignItems: "center",
+            padding: "10px 15px",
+            cursor: "pointer",
+            display: "flex",
+            border: `1px solid ${theme.palette.grey[100]}`,
+            textTransform: "initial",
+            fontWeight: theme.typography.fontWeightRegular,
+
+            "& [class*='MuiSvgIcon-root']": {
+                fontSize: 12,
+            },
+
+            "&:active": {
+                border: `1px solid ${theme.palette.grey[400]}`,
```

Comment:
`1px` and `solid` are already set in the root.  
Overriding only the color would make it easer to customize the border-width for example, currently you would need to override the width for multiple selectors. When you only change the border-color, it would be enough to override the border-width of root. 

```suggestion
                border-color: theme.palette.grey[400],
```